Sources/Sentry/SentryScope.m

```diff
@@ -87,17 +92,30 @@ - (instancetype)init
 - (instancetype)initWithScope:(SentryScope *)scope
 {
     if (self = [self init]) {
-        self.extraDictionary = scope.extraDictionary.mutableCopy;
-        self.tagDictionary = scope.tagDictionary.mutableCopy;
+        @synchronized(_extraDictionary) {
```

Comment:
Do we need to synchronize on `init`? Isn't this executed before anyone has a reference to this object?

Reply:
Good point, we don't need it. I removed the synchronized in here.